DFC.ServiceTaxonomy.ApiFunction/Models/Cypher.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DFC.ServiceTaxonomy.ApiFunction.Models
+{
+    public class Cypher
```

Comment:
This class looks semantically equivalent to the driver's `Statement` class, so it's probably worth using that instead,

Reply:
Actually, it's not equivalent, as it's specifying the required params, rather than actually supplying the params.